src/bramble/client/ProjectStats.js

```diff
@@ -0,0 +1,258 @@
+define([
+    // Change this to filer vs. filer.min if you need to debug Filer
+    "thirdparty/filer/dist/filer.min",
+],function(Filer){
+    "use strict";
+
+    var ProjectStats = {};
+    
+    var _fs = new Filer.FileSystem();
+    ProjectStats.getFileSystem = function(){
+        if(_fs){
+            return _fs;
+        }
+        return null;
+    };
+
+    var _shell = new _fs.Shell();
+    var _Path = Filer.Path;
+    var _root;
```

Comment:
`var _shell = new _fs.Shell();` on line 17 is supposed to go after the _fs declaration or it will give an error.